grayskull/base/base_recipe.py

```diff
@@ -178,9 +178,26 @@ def generate_recipe(
 
         :param folder_path: Path to the folder
         """
-        recipe_dir = Path(folder_path) / self.get_var_content(
+        print(f"This is the folder_path: {folder_path}")
+        package_name = self["package"]["name"].values[0]
+        print(f"This is the package name before split: {package_name}")
+        if f"{package_name}".startswith(("http://", "https://")):
```

Comment:
Similar feedback here (or create a `utils.something` function):

In `grayskull.utils`:

```python
def origin_is_github(name_or_url):
    return name_or_url.startswith(("http://", "https://"))
```